sklearn/utils/testing.py

```diff
@@ -577,9 +577,38 @@ def uninstall_mldata_mock():
     datasets.mldata.urlopen = urlopen
 
 
-def all_estimators(include_meta_estimators=None,
-                   include_other=None, type_filter=None,
-                   include_dont_test=None):
+# Meta estimators need another estimator to be instantiated.
+META_ESTIMATORS = ["OneVsOneClassifier", "MultiOutputEstimator",
+                   "MultiOutputRegressor", "MultiOutputClassifier",
+                   "OutputCodeClassifier", "OneVsRestClassifier",
+                   "RFE", "RFECV", "BaseEnsemble", "ClassifierChain",
+                   "RegressorChain", "SelfTrainingClassifier"]
+# estimators that there is no way to default-construct sensibly
+OTHER = ["Pipeline", "FeatureUnion",
+         "GridSearchCV", "RandomizedSearchCV",
+         "SelectFromModel", "ColumnTransformer"]
+
+# some strange ones
+DONT_TEST = ['SparseCoder', 'DictVectorizer',
+             'LabelBinarizer', 'LabelEncoder',
+             'MultiLabelBinarizer', 'TfidfTransformer',
+             'TfidfVectorizer', 'IsotonicRegression',
+             'OneHotEncoder', 'RandomTreesEmbedding', 'OrdinalEncoder',
+             'FeatureHasher', 'DummyClassifier', 'DummyRegressor',
+             'TruncatedSVD', 'PolynomialFeatures',
+             'GaussianRandomProjectionHash', 'HashingVectorizer',
+             'CheckingClassifier', 'PatchExtractor', 'CountVectorizer',
+             # GradientBoosting base estimators, maybe should
+             # exclude them in another way
+             'ZeroEstimator', 'ScaledLogOddsEstimator',
+             'QuantileEstimator', 'MeanEstimator',
+             'LogOddsEstimator', 'PriorProbabilityEstimator',
+             '_SigmoidCalibration', 'VotingClassifier']
+
+
+def all_estimators(include_meta_estimators=False,
```

Comment:
Was this change to get a test to work?

Reply:
This looks like a merge error, I'll remove it and see what CI says.